vmc/resource_vmc_cluster.go

```diff
@@ -368,18 +379,19 @@ func resourceClusterUpdate(d *schema.ResourceData, m interface{}) error {
 			task, err := taskClient.Get(orgID, task.Id)
 			if err != nil {
 				if err.Error() == (errors.Unauthenticated{}.Error()) {
-					log.Print("Auth error", err.Error(), errors.Unauthenticated{}.Error())
+					log.Printf("Authentication error : %v", errors.Unauthenticated{}.Error())
 					err = connectorWrapper.authenticate()
 					if err != nil {
 						return resource.NonRetryableError(fmt.Errorf("authentication error from Cloud Service Provider : %s", err))
 					}
 					return resource.RetryableError(fmt.Errorf("instance update still in progress"))
 				}
-				return resource.NonRetryableError(fmt.Errorf("error describing instance: %s", err))
-
+				return resource.NonRetryableError(fmt.Errorf("error updating EDRS policy configuration : %v", err))
 			}
-			if *task.Status != "FINISHED" {
-				return resource.RetryableError(fmt.Errorf("expected instance to be updated but was in state %s", *task.Status))
+			if *task.Status == "FAILED" {
+				return resource.NonRetryableError(fmt.Errorf("task failed to EDRS policy configuration"))
```

Comment:
an action verb is required in this message Eg: update

Reply:
Done